lib/vmpooler/vsphere_helper.rb

```diff
@@ -196,39 +168,96 @@ def find_folder(foldername)
       base
     end
 
-    def find_least_used_host(cluster)
+    def get_host_utilization(host, model=nil, limit=90)
+      # Returns an array containing cumulative CPU and memory utilization of a host, and its object reference
+      # Params:
+      # +model+:: CPU arch version to match on
+      # +limit+:: Hard limit for CPU or memory utilization beyond which a host is excluded for deployments
```

Comment:
This should docstring should go above the method signature in order for it to actually be a docstring.  Down here it's just a comment at the top of the body.